compatibility/bazel_tools/testing.bzl

```diff
@@ -128,7 +69,7 @@ excluded_test_tool_tests = {
             "exclusions": ["ContractKeysIT"],
```

Comment:
As this version is not going to be tested anymore, should we remove this exclusion?

Reply:
I don’t feel too strongly about this but it seems easier to me to just keep the exclusions. They don’t cause any harm.

Reply:
Otherwise you probably also want to switch all the other bounds to start or end at versions that we include. That just seems more tricky to keep in sync.